src/assets/styles/50-utilities/_scaffolding.scss

```diff
@@ -40,3 +40,9 @@
     @include grid-column(1, 13);
   }
 }
+
+.u-scaffold-grid-item--extended {
+  @include mq(md) {
+    @include grid-column(4, 13);
+  }
+}
```

Comment:
I'm very surprised to find these styles in corporate-components. I feel they should be declared in corporate-react as the component they apply to only exists in corporate-react.

What do you think @creido-welly?

Reply:
Yes it makes sense to move them